docs/source/upgrade-guide/index.md

```diff
@@ -37,6 +37,23 @@ NodeJS 12 is deprecated in Volto 13.
 Please update your projects to a NodeJS LTS version, where either 14 or 16 is supported at the moment of this writing.
 Version 16 is recommended.
 
+### Removed `date-fns` from build
+
+The `date-fns` library has been removed from Volto's dependencies.
+It was in the build because `Cypress` depended on it.
+After the `Cypress` upgrade it no longer depends on it.
+If your project still depends on it, add it as a dependency of your project.
+
+```{warning}
+The `date-fns` version present in the build was quite an old one (1.x.x series). Beware to use an updated version (2.x.x), since it may contain quite some breaking changes. Also take a look at: https://dockyard.com/blog/2020/02/14/you-probably-don-t-need-moment-js-anymore
```

Comment:
```suggestion
The `date-fns` version present in the build was quite old (1.x.x series).
Beware when using an updated version (2.x.x), as it may contain some breaking changes.
Also take a look at: https://dockyard.com/blog/2020/02/14/you-probably-don-t-need-moment-js-anymore
```